components/profile-controller/config/manager/kustomization.yaml

```diff
@@ -3,9 +3,9 @@ resources:
 - service-account.yaml
 
 configMapGenerator:
-- name: config
-  literals:
+- literals:
   - ADMIN=
   - WORKLOAD_IDENTITY=
   - USERID_HEADER="kubeflow-userid"
   - USERID_PREFIX=
+  name: config
```

Comment:
:newline char needed

Reply:
this was cherry-picked as-is from the upstream commit, generally, I would agree, but in this situation I would propose to keep the changeset as small as possible

Reply:
makes sense 👍